README.md

```diff
@@ -32,7 +32,7 @@ Building Scylla with the frozen toolchain `dbuild` is as easy as:
 ```bash
 $ git submodule update --init --force --recursive
 $ ./tools/toolchain/dbuild ./configure.py
-$ ./tools/toolchain/dbuild ninja build/release/scylla
+$ ./tools/toolchain/dbuild ninja -C build scylla:RelWithDebInfo
```

Comment:
I saw that `ninja -C build Dev/scylla` works, so why write `scylla:Dev` with a colon? The benefit of `Dev/scylla` is that it's the actual path of build/Dev/scylla, which you'll probably need to know.
Also, what the heck is "RelWithDebInfo" and is this really the simplest example we can give?

Reply:
i am using `ninja -C build Dev/scylla` in the latest revision. "RelWithDebInfo" is one of the typical `CMAKE_BUILD_TYPE`. it is the closest thing to "release". see https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html